custom/icds_reports/mobile_views.py

```diff
@@ -67,8 +72,25 @@ class MobileDashboardView(TemplateView):
     def domain(self):
         return self.kwargs['domain']
 
+    def _has_helpdesk_role(self):
```

Comment:
this was just copied form web view, right? can we move it (and the `report_an_issue_url` param addition) to `dashboard_utils`?

Reply:
Sure. I will make the change